packages/prediction-app/src/features/settings/ChapSettings/hooks/useChapStatus.ts

```diff
@@ -3,18 +3,19 @@ import { Route } from "../../../../hooks/useRoute";
 import { ApiError, DefaultService } from "@dhis2-chap/chap-lib";
 
 type Props = {
-    route: Route
+    route: Route | undefined
 }
 
 export const useChapStatus = ({ route }: Props) => {
     const { data: status, error, isLoading } = useQuery<Awaited<ReturnType<typeof DefaultService.systemInfoSystemInfoGet>>, ApiError>({
-        queryKey: ['systemInfo', route.url],
+        queryKey: ['systemInfo', route?.url],
         queryFn: () => DefaultService.systemInfoSystemInfoGet(),
+        enabled: !!route,
     })
```

Comment:
Let's also add 5 minutes of caching. (Both stale time and cash time). Also set retries to zero so that it doesn't try three times before throwing an error.